airbyte-integrations/connectors/source-lever-hiring/unit_tests/conftest.py

```diff
@@ -6,18 +6,29 @@
 
 
 @fixture
-def test_config():
-    return {
-        "credentials": {
-            "client_id": "test_client_id",
-            "client_secret": "test_client_secret",
-            "refresh_token": "test_refresh_token",
-            "access_token": "test_access_token",
-            "expires_in": 3600,
-        },
-        "environment": "Sandbox",
-        "start_date": "2021-05-07T00:00:00Z",
-    }
+def test_config(auth_type):
+    if auth_type == "Client":
+        return {
+            "credentials": {
+                "client_id": "test_client_id",
```

Comment:
`"auth_type": "Client",`

do you need to add an `auth_type`? same below for `elif`

Reply:
Not necessarily, it was just to be able to perform the unit tests with the same function, but frankly I'd rather not use conftest.py file at all here and have everything in the same file, like you suggested